.travis.yml

```diff
@@ -1,43 +1,62 @@
-sudo: required
-services:
-- xvfb
-notifications:
-  email: false
-cache:
-- ccache
+# This travis image builds roboteam_suite and runs the target 'ai_tests'.
+# All other repositories of roboteam_suit are set to the master branch.
+# We need all dependencies to parse the cmake files. This can be optimized later,
+# but with caching this is not a large performance hit anyway.
 
 language: cpp
-matrix:
-  include:
-  - name: "Xenial"
-    dist: xenial
+os: osx
+osx_image: xcode11.2
 
-before_install:
-- sudo apt-get clean
-- sudo apt update
-- sudo apt-get install -y ccache ninja-build python3 python3-pip autoconf automake libtool make g++ unzip lcov gcovr qtbase5-dev libzmq3-dev libzmqpp-dev libarmadillo-dev
```

Comment:
Not all dependencies seem to be transferred from ubuntu to osx, but most of them are standard on the travis osx version as well, except possibly the code coverage ones?

Reply:
But as you mention, we should probably skip code cov for now